aibolit/ml_pipeline/ml_pipeline.py

```diff
@@ -11,16 +11,25 @@
 from aibolit.config import CONFIG
 
 
+JAVA_FILES_PATH = os.environ['JAVA_FILES_PATH']
+
 def collect_dataset():
```

Comment:
collect _dataset now has a parameter with java folder, no need to use os.environ. Fix it